src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -121,7 +121,7 @@ internal sealed partial class TexlBinding
 
         public bool HasLocalScopeReferences { get; private set; }
 
-        public ErrorContainer ErrorContainer { get; } = new ErrorContainer();
+        public ErrorContainer ErrorContainer { get; set; } = new ErrorContainer();
```

Comment:
Instead of making this mutable, couldn't the error container instead be passed as an arg to the delegation strategy checkers, so that it puts errors there rather than using the binder's error container? I agree with the concerns others have raised here

Reply:
The issue is that the delegation strategy checkers delegate their functionality to other methods other objects (e.g. IsRowScopedServerDelegatable), which would also need to take an error container instead of using the one attached to the binder.  This would introduce additional fragility.

Reply:
Is it possible for multiple methods or objects to modify the mutable property? If so, how can you guarantee that the error container behavior returns to the way it was before it was modified? For example, is there ever a case where there should be a stack of error containers instead of a mutable property?

Reply:
I've replaced the mutable property with a specific override method, to attempt to limit misuse.  I've also added a unit test to cover nested scenarios.